enum.rb

```diff
@@ -0,0 +1,53 @@
+#:nodoc:
+module Enumerable
+  def my_each
+    length.times { |index| yield(self[index]) }
+    self
+  end
+
+  def my_each_with_index
+    length.times { |index| yield(self[index], index) }
+    self
+  end
+
+  def my_select
+    array = []
+    my_each { |index| array << index if yield(index) }
+    array
+  end
+
+  def my_all?
+    new_array = true
+    my_each { |index| break new_array = false unless yield(index) }
+    new_array
+  end
+
+  def my_any?
+    new_array = false
+    my_each { |index| break new_array = true if yield(index) }
+    new_array
+  end
+
+  def my_none?
+    new_array = true
```

Comment:
### Should have working `my_none?`identical and returns the same thing as ruby's [`none?`](https://ruby-doc.org/core-2.6.4/Enumerable.html#method-i-none-3F). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_enumerable/my_none.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-2))
``` p %w{ant bear cat}.my_none?(/d/)``` should return same results as ``` p %w{ant bear cat}.none?(/d/)```
``` p [1, 3.14, 42].my_none?(Float)``` should return same results as ```[1, 3.14, 42].none?(Float)```
``` p [nil].my_none?``` should return same results as ``` p [nil].none?```
``` p [nil, false].my_none? ``` should return same results as ``` p [nil, false].none? ``` 
``` p [nil, false, true].my_none? ``` should return same results as ``` p [nil, false, true].none? ```